boards/arm/colibri_imx7d_m4/pinmux.c

```diff
@@ -1,191 +0,0 @@
-/*
- * Copyright (c) 2018, Diego Sueiro <diego.sueiro@gmail.com>
- *
- * SPDX-License-Identifier: Apache-2.0
- */
-
-#include <init.h>
-#include "device_imx.h"
-
-static int colibri_imx7d_m4_pinmux_init(const struct device *dev)
```

Comment:
note: this commit deletes `pinmux.c`, but I can;t see any equivalent `board-pinctrl.dtsi` being introduced.